app/views/users/show.html.slim

```diff
@@ -92,7 +92,7 @@
   hr
 
   .col-sm-12
-    .row.card
+    .row.card.d-block
```

Comment:
For some reason there is a large padding at the bottom of the card after expanding an order in the user history overview. Changing the card from a flexbox to a block fixes this somehow. If anybody can find a better fix for or the origin of this issue, be my guest.

Reply:
Als je de table de class `mb-0` geeft, en de `p-2` class van `tab-content` weghaalt, dan heb je hetzelfde effect.